gustaf/utils/__init__.py

```diff
@@ -1,4 +1,5 @@
 from gustaf.utils import arr, connec, log, tictoc
```

Comment:
do you need to keep the direct tictoc import in this line? if you add a renamed one in the next line?

Is this for historical reasons? should we add a deprecation warning?

Reply:
yes